### PR TITLE
[MIG][13.0] partner_delivery_schedule: Migration to V13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT21575
@carlosdauden @Tardo Review please

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: partner_delivery_schedule/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+To use this module you need to:
+
+#. Go to *Contacts > Create* or *Sales > Customers > Create*.
```
Comment (Tardo) [inline partner_delivery_schedule/readme/USAGE.rst]:
```suggestion
#. Go to *Contacts > Create* or *Sales > Orders > Customers > Create*.
```

File: partner_delivery_schedule/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+#. Go to *Sales and Purchases* tab.
+#. Create new delivery hours for this partner in *Delivery Schedule* field.
+
+You can set deliveries schedule directly in
+*Sales > Configuration > Delivery Hours*
```
Comment (Tardo) [inline partner_delivery_schedule/readme/USAGE.rst]:
```suggestion
*Sales > Configuration > Delivery Schedule*
```

**Review by Tardo — CHANGES_REQUESTED** [review]
:+1: LGTM only documentation changes

**Comment by sergio-teruel** [comment]
@Tardo Changes done

**Review by carlosdauden — APPROVED** [review]
Code LGTM
I don't understand travis precommit error...

**Pull request merged**